CONTRIBUTING.rst

```diff
@@ -29,13 +29,28 @@ We use flake8 to check some PEP8 errors/warnings. This will be checked with
 Simple Bandwidth Scanner is moving towards 100% test coverage. Please help us
 reach that goal, or at least not move us away from it.
 
+Git and GitHub Guidelines
+=========================
+
 **Write good commit messages** that at least follow the spirit of
 :ref:`this example <commit-msg>`.
 
-**Write many small commits** each containing an atomic change instead of one
-large mega-commit. This not only makes code review easier, but it also makes
-commits that show up in ``git blame`` 10 years from now make more sense.
+Strive to **write many small commits** each containing an atomic change instead
+of one large mega-commit. This not only makes code review easier, but it also
+makes commits that show up in ``git blame`` 10 years from now make more sense.
+
+**Prefer a rebase workflow instead of merge**. Incorporating PRs should be done
```

Comment:
Rebase workflow makes commits cleaner, however, unless done in different branches as you commente below, it would imply "lying" for remote branches.